network/server.go

```diff
@@ -181,6 +182,11 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 	logger.Info("LibP2P server running", "addr", AddrInfoToString(srv.AddrInfo()))
 
 	if !config.NoDiscover {
+
+		if config.Chain.Bootnodes != nil && len(config.Chain.Bootnodes) < 2 {
```

Comment:
This is a breaking change - please check the PR box that this is a breaking change, and in the breaking changes section explain why this will cause clusters with 1 bootnode to halt

Reply:
Done